source/Calamari/Program.cs

```diff
@@ -81,6 +81,13 @@ protected override void ConfigureContainer(ContainerBuilder builder, CommonOptio
             builder.RegisterType<KubectlGet>().As<IKubectlGet>().SingleInstance();
             builder.RegisterType<HelmTemplateValueSourcesParser>().AsSelf().SingleInstance();
 
+            
+#if WINDOWS_CERTIFICATE_STORE_SUPPORT 
```

Comment:
Eventually we will get rid of this compiler directive entirely, until the cert logic is externalized, this allows the command classes above to take a dependency on `IWindowsX509CertificateStore`, even though they should never end up invoking it.